src/components/ClassInfoCard.tsx

```diff
@@ -27,8 +29,10 @@ type ClassInfoProps = {
  */
 export const ClassInfoCard: React.FC<ClassInfoProps> = ({
     cardInfo,
+    isLegible,
```

Comment:
Can you explain what `isLegible` is for? When is it false (not readable). A bit confused on what readability has to do with the context

Reply:
yea should be `eligible` mb